src/InsertionsClient.Console/ConsoleApp/Program.cs

```diff
@@ -230,6 +228,7 @@ private static void ShowHelp()
             Trace.WriteLine($"{Environment.NewLine}Options:");
             Trace.WriteLine($"{SwitchDefaultConfig}   full path on disk to default.config to update");
             Trace.WriteLine($"{SwitchManifest}   full path on disk to manifest.json");
+            Trace.WriteLine($"{SwitchWhitelistedPackages}   full path on disk to whitelisted packages file. Each line should have a package id. Regex is also supported [optional]");
```

Comment:
super nit: `Regex is also supported` is a bit misleading because it's always parsed as a regex and the user has to be aware of that so they properly escape special chars even if they are not interested in using any regex functionality.

Reply:
Agreed, rephrasing.